homeworks/artem.sokur_Grimdvl/PokemonProject/index.html

```diff
@@ -44,30 +47,30 @@ <h1 class="about__text__title">About me</h1>
                         <br>
                         This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to create my own small project for the portfolio.<br>
                         <br>
-                        You can contact me via <a class="about__contact__telegram" href="https://t.me/Grim_679">Telegram</a>
-                        and/or check out my <a class="about__contact__github" href="https://github.com/Grimdvl">GitHub</a>.
+                        You can contact me via <a class="about__contact-telegram" href="https://t.me/Grim_679">Telegram</a>
+                        and/or check out my <a class="about__contact-github" href="https://github.com/Grimdvl">GitHub</a>.
```

Comment:
Better use styles that you used for MastersAcademy site `class="about__contact"`